File: java_samples/payloads/merge_fields.java
package payloads;

import org.json.JSONObject;

public class merge_fields {
	private String label;
	private String value;
	private int font_size = 0;

	public merge_fields(String label, String value, int font_size) 
	{
		this.label = label;
		this.value = value;
		this.font_size  = font_size;
	}
	
	public merge_fields(String label, String value)
	{
		this.label = label;
		this.value = value;
	}
	
	
	public JSONObject toJson() 
	{
		JSONObject obj = new JSONObject();

		obj.put("label", this.label);
		obj.put("value", this.value);
		obj.put("font_size", this.font_size);
		return obj;
		
	}

}

File: node_sample/payload/MergeFields.js

class MergeFields{
    label = ""
    value = ""
    font = 0

    constructor(label, value, font=0)
  {
    this.label = label;
    this.value = value;
    this.font = font
  }

  to_json(){
  return {"label": this.label, "value": this.value, "font": this.font}
  }
}

module.exports = MergeFields